classes/plugins/Plugin.inc.php

```diff
@@ -564,29 +572,28 @@ function installContextSpecificSettings($hookName, $args) {
 		// install context specific settings.
 		$application = Application::getApplication();
 		$contextDepth = $application->getContextDepth();
-		if ($contextDepth > 0) {
```

Comment:
FYI, a lot of this was from the days where we maintained OCS (2 levels of "context depth") and OHS (0 levels). I don't see any need to maintain those distinctions anymore.